nucypher/policy/conditions/types.py

```diff
@@ -75,6 +75,12 @@ class JsonRpcConditionDict(BaseExecConditionDict):
     authorizationToken: NotRequired[str]
 
 
+class JWTConditionDict(_AccessControlCondition):
+    jwt_token: str
+    public_key: str  # TODO: See #3572 for a discussion about deprecating this in favour of the expected issuer
+    expected_issuer: NotRequired[str]
```

Comment:
This is for the JSON itself (before it gets converted to snake case) so it should be camelcase instead.

```suggestion
    jwtToken: str
    publicKey: str  # TODO: See #3572 for a discussion about deprecating this in favour of the expected issuer
    expectedIssuer: NotRequired[str]
```